data/remnant missions.txt

```diff
@@ -493,5 +558,748 @@ mission "Remnant: Technology Available"
+			`As you step out into the icy chill of Viminal you get a comlink message from Tali asking if you could meet her in the cafeteria. Replying that you can, you make your way through the crowds and find an empty table. Moments later Tali appears and grabs two trays from the dispenser before heading towards you. She settles down in the opposite chair with a look of exhaustion.`
+			`	"It is good to see you again, <first>," she sings softly. When you sign a reply she sits up and smiles. "My, you are a quick learner," she signs. "Great, this is much faster."`
+			`	"I haven't fully..." (She makes a sign that you don't understand, but seems similar to the signs for "break" and "understand".) "...the technology you retrieved for us, but we should be ready to produce them soon. In the meantime, though, they are showing us that humanity has made significant progress since we departed. Could you help us catch up with what has happened out there?"`
+			choice
+				`	"Yes."`
+				`	"Sorry, not interested."`
+					decline
+			`	"Excellent! According to a friend of mine who is versed in history from the war, there were rumors that someone was building a giant library out in the Deep. If it still exists, that would likely be where to go."`
+				accept
+	on stopover
+		conversation
+			`After having traveled clear across the galaxy, you finally have arrived at the archives of human history. On the direction of the librarians you locate the gift shop and buy a "souvenir copy" of the archive. You idly wonder if you should buy a couple spare copies just in case, but decide against it.` 
+	
+	on complete
+		conversation
+			`Having crossed the galaxy for a copy of the archive, you gently set <ship> down on Viminal hoping that this information is helpful. The port seems unusually busy, but the comm channels are quiet beyond the bare minimum to keep traffic organized.`
+			`	Before you have even taken the time to shutdown the engines Tali is halfway up the side of your ship securing a refueling hose and charging cable. As you reach for the comlink she scrambles up to the bridge windows and quickly rattles off a series of signs at you. All you can make out is "Keep...hot, incoming something!" Before you can frame a reply she has raced across your wing and leapt to the Albatross parked next to you.`
+			`	Looking around more carefully, you note that every ship in the port is idling with fuel lines still connected. If this were some Republic port the safety inspectors would be having a nightmare. One Starling is getting holes in its wings patched even as the crew appear to be running prep-for-flight checklists. It also looks like every ship has a guard. Something tells you that right now would be a bad time to take a walk around town.`
+
+mission "Remnant: Salvage 2"
+	landing
+	source "Viminal"
+	name "Defend <planet>"
+	description "Assist in defending <planet> from a Korath raid, then return to the planet."
+	to offer
+		has "Remnant: Salvage 1: done"
+	on offer
```

Comment:
I'm a bit confused: I should move the `on complete` conversation from Salvage 1 to the `on offer` for salvage 2 because currently salvage 2's `on offer` appears after salvage 1's `on complete`? Unless I'm mis-understanding something, that sounds like "working as designed."

Reply:
The first conversation might as well be merged with the second is what I’m saying, as it’s a single fluid moment. The only other instance in game that I can recall where an on complete conversation forces right into the landing on offer of another mission is when you’re knocked unconscious by Republic Intelligence, in which case this disjointed conversation flow works, but that type of flow makes no sense here.

Reply:
Even with the break in flow? It doesn't specify how long the player sits around waiting for something to happen. If I put it all in one conversation, I'd be strongly inclined to put in a 
```-----
A tense half an hour later...
-----"```
block/interruption in the middle of it to separate the two. It feels like the sort of place one would have a break in the continuity.

Reply:
I definitely would not put such a block into a conversation, certainly not with the dashes, because that creates a *huge* clash with Endless Sky's narrational style.